cmd/hub.go

```diff
@@ -8,9 +8,9 @@ import (
 
 // hubDesc is the description of the hub command
 const hubDesc = `
-Download an experiment chart to a local directory.
+Download Iter8 experiment charts from a Git repo.
```

Comment:
Are we restricted to a Git repo? My impression from the code is that it could be from any of many places (there is a list of getters used by Get()).